app/views/users/notes/_search.html.slim

```diff
@@ -0,0 +1,9 @@
+.top-bar
+  = simple_form_for :search_form,
+                    method: :get, url: action,
+                    html: { class: "menu" } do |f|
+    li
+      = f.input :query, label: false, placeholder: "Search notes",
+                input_html: { value: params.fetch(:search_form, {})[:query] }
+    li
+      = f.submit :Search, class: "button"
```

Comment:
Why do you need a symbol with a capital letter here? I think it will be better to use a string here.

```suggestion
      = f.submit "Search", class: "button"
```